scan/context.go

```diff
@@ -185,12 +176,12 @@ func getWithStatusError(url string) (resp *http.Response, err error) {
 		// If the status code is 4xx then read the body and return an error
 		// If the status code is a 5xx then check the body on the final attempt and return an error.
 		if resp.StatusCode < 500 || attempt == maxRetries-1 {
-			body, bodyReadErr := ioutil.ReadAll(resp.Body)
```

Comment:
`ioutil` is deprecated as of Go 1.16, `io` should be preferred for the same functionality.